make/stratos/_env

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+git_root=$(git rev-parse --show-toplevel)
+. "${git_root}/.envrc"
+
+: "${STRATOS_URL:=https://github.com/cloudfoundry-incubator/stratos/releases/download/2.1.1/console-helm-chart-2.1.1-74090ba.tgz}"
+: "${STRATOS_NAME:=stratos}"
```

Comment:
Is this the helm release name? The name here is kind of generic.

Reply:
This is the Helm installation name.

Reply:
I think (for reasons I don't know) helm wants to call that `release`. For example the [docs for `helm delete`](https://docs.helm.sh/helm/#helm-delete) say "given a release name".